### PR TITLE
ES-851869- FAQ added in Presentation product.

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
FAQ added in Presentation product.

TaskLink: <a href="https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/851869">Why Metafile images doesn't gets preserved in Presentation to PDF/Image conversion</a>

Regards,
Venkatesh

### CODE REVIEW

File: File-Formats/Presentation/FAQ.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ Current version of the Essential Presentation library does not provide support f
 
 You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
 
+## Why Metafile images doesn't gets preserved in Presentation to PDF/Image conversion?
+
+In .NET Core targeting applications, metafile images such as EMF and WMF have some limitations. Presentation library used SkiaSharp graphics library in Syncfusion.PresentationRenderer.Net.Core to render the text and images in PDF/Image conversion. But SkiaSharp doesn’t support the metafile images such as EMF and WMF format images, so, it doesn’t preserve the EMF images in ASP.NET Core platform. So, those images will not preserve in Presentation document to Image/PDF conversion using Presentation library.
```
Comment (florenceliliana) [inline File-Formats/Presentation/FAQ.md:53]: In .NET Core targeting applications, metafile images such as EMF and WMF have some limitations. The presentation library used SkiaSharp graphics library in Syncfusion.PresentationRenderer.Net.Core to render the text and images in PDF/Image conversion. But SkiaSharp does not support metafile images such as EMF and WMF format images, so it does not preserve the EMF images in the ASP.NET Core platform. So, those images will not be preserved in the Presentation document to Image/PDF conversion using the Presentation library.

File: File-Formats/Presentation/FAQ.md [hunk 2]
```diff
@@ -48,6 +48,12 @@ Current version of the Essential Presentation library does not provide support f
...
+
+As a work around to preserve the expected images in the converted Image/PDF, we suggest you to replace the EMF image to its equivalent JPEG or PNG and then convert the Presentation document to Image/PDF in order to preserve the image in the Image/PDF.
```
Comment (florenceliliana) [inline File-Formats/Presentation/FAQ.md:55]: As a workaround to preserve the expected images in the converted Image/PDF, we suggest you replace the EMF image with its equivalent JPEG or PNG and then convert the Presentation document to Image/PDF in order to keep the image in the Image/PDF.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The content review is completed.

File: File-Formats/Presentation/FAQ.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/Presentation/FAQ.md]: Modified

File: File-Formats/Presentation/FAQ.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/Presentation/FAQ.md]: Modified

**Pull request closed without merging**